src/content/en/2022/pwa.md

```diff
@@ -1,22 +1,525 @@
+- `sync`, which fires when the UA believes the user has connectivity, has a usage of 6% on desktop and 5% on mobile.
+- `periodicsync`, which allows web applications to periodically synchronize data in the background, is currently at 0.01% on both desktop and mobile platforms. It should be noted that `periodicsync` is limited to a max of once every 12 hours. This can be artificially suppressing usage of the feature.
+
+### Other popular SW features
+
+{{ figure_markup(
+  image="usage-skip-waiting.png",
+  caption="Usage of `skipWaiting()` method.",
+  description="Column chart showing the usage of the `skipWaiting()` method is 60% on desktop and 59% on mobile",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vS7KgpDrr_m20ZDaHDBYLFSGNVdj3lhGHbHopEi4071q21U5rZDJfbYrrdyaEIU1D9cwgxxBCI6NBsV/pubchart?oid=174072819&format=interactive",
+  sheets_gid="58784102",
+  sql_file="sw_methods.sql"
+  )
+}}
+
+Similar to the [stats of last year](../2021/pwa#other-popular-service-worker-features), the [`skipWaiting`](https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerGlobalScope/skipWaiting) method that is used to immediately activate the service worker is still very popular among developers, being present on 60% of desktop and 59% of mobile web apps.
+
+These are the top most used service worker objects:
+
+{{ figure_markup(
+  image="most-used-sw-objects.png",
+  caption="Most used service worker objects.",
+  description="Column chart showing `clients` objects are used on 93% of desktop and 87% of mobile websites, `caches` on 45% and 44% respectively, `cache` on 21% and 21%, and finally `client` on 12% of desktop and 13% mobile PWA websites.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vS7KgpDrr_m20ZDaHDBYLFSGNVdj3lhGHbHopEi4071q21U5rZDJfbYrrdyaEIU1D9cwgxxBCI6NBsV/pubchart?oid=1869777638&format=interactive",
+  sheets_gid="1746822463",
+  sql_file="sw_objects.sql"
+  )
+}}
+
+{# TODO Anything to say about this graph? #}
+
+And these are the most used methods:
+
+{{ figure_markup(
+  image="most-used-sw-objects-methods.png",
+  caption="Most used service worker object methods.",
+  description="Column chart showing `clients.matchAll` is used on 61% of desktop and 55% of mobile PWA sites, `clients.claim` on 58% and 58% respectively, `clients.openWindow` on 56% and 51%, `caches.open` on 44% and 43%, `caches.delete` on 29% and 24%, `caches.match` on 25% and 28%, `caches.keys` on 21% and 18%, `cache.put` on 12% and 14%, `client.url` on 10% and 11%, and finally `cache.add` is used on 8% of desktop and 7% of mobile PWA sites.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vS7KgpDrr_m20ZDaHDBYLFSGNVdj3lhGHbHopEi4071q21U5rZDJfbYrrdyaEIU1D9cwgxxBCI6NBsV/pubchart?oid=1209078963&format=interactive",
+  sheets_gid="1969934262",
+  sql_file="sw_methods.sql"
+  )
+}}
+
+{# TODO Anything to say about this graph? #}
```

Comment:
@diekus anything we could add to this rather than just giving the graph with no commentary?